jcli/src/jcli_app/certificate/get_stake_pool_id.rs

```diff
@@ -9,7 +9,7 @@ use structopt::StructOpt;
 pub struct GetStakePoolId {
     /// read the certificate from
     pub input: Option<PathBuf>,
-    /// write the certificate too
+    /// write the id to
```

Comment:
while I appreciate the fix, it would be better split into a separate commit. Otherwise the git history will show that this was changed for adding the `get-vote-plan-id` while it is actually a wording change in the doc